help/develop/javascript-development/debugging.md

```diff
@@ -0,0 +1,103 @@
+# Debugging
+
+This document discusses debugging of your packages for Datagrok in two popular IDEs &mdash; Visual Studio Code and 
+JetBrains WebStorm.
+
+## One-click debugging with Visual Studio Code
+
+To configure VS Code for Datagrok development, run the `grok create` command with the `--ide=vscode` flag:
+
+```shell
+grok create my-datagrok-package --ide=vscode
+```
+
+**IMPORTANT**: The `--ide` flag only works on Windows. If you're using Linux or macOS, refer to the
+[Creating a VS Code Configuration for Linux or macOS section].
```

Comment:
We definitely don't want to force people to create these files manually :) I hope to add the working version for these OS this week.